.github/workflows/publish-technical-documentation-release.yml

```diff
@@ -0,0 +1,74 @@
+name: publish-technical-documentation-release
+
+on:
+  push:
+    branches:
+      - <BRANCH GLOB>
+    tags:
+      - <TAG GLOB>
+    paths:
+      - "docs/sources/**"
+  workflow_dispatch:
+jobs:
+  sync:
+    if: github.repository == 'grafana/grafana'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Grafana repo
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Checkout Actions library
+        uses: actions/checkout@v4
+        with:
+          repository: grafana/grafana-github-actions
+          path: ./actions
+
+      - name: Install Actions from library
+        run: npm install --production --prefix ./actions
+
+      - name: Determine if there is a matching release tag
+        id: has-matching-release-tag
+        uses: ./actions/has-matching-release-tag
+        with:
+          ref_name: ${{ github.ref_name }}
+          release_tag_regexp: <TAG REGEXP>
+          release_branch_regexp: <BRANCH REGEXP>
```

Comment:
As for the globs, I'd use the same configuration as the Grafana Agent:
```
        release_tag_regexp: "^v(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)$"
        release_branch_regexp: "^release-v(0|[1-9]\\d*)\\.(0|[1-9]\\d*)$"
```

Reply:
Thanks, added in c5d68ec863d90cca13e8612ff8f76fbc3fdcee46